### DISCUSSION TITLE
Adguard vpn CLI support

### DISCUSSION THREAD

**Discussion by wocsom92** [RaspAP/raspap-webgui#1687] [Ideas]
What about extending the VPN provider CLI support by Adblock VPN?

**Comment by billz** [comment]
Currently in progress https://github.com/RaspAP/raspap-webgui/issues/1705

<img width="426" alt="adguard" src="https://github.com/user-attachments/assets/46ba1d4e-8b99-4204-a104-ff23c30471b5">

**Comment by billz** [comment]
This is available in the [latest release](https://github.com/RaspAP/raspap-webgui/releases/tag/3.2.2). If you're using RaspAP's 'Check for update' function on the **About** page, this will update the available providers and supporting code. However, this method doesn't run the installer interactively to specify your desired provider. In this event, you can enable AdGuard by executing `sudo nano /etc/raspap/provider.ini` and setting the following value:

```
providerID = 4
```

Save and exit this file, then refresh the web UI.
